udata/migrations/2017-07-07-deduplicate-org-members.js

```diff
@@ -0,0 +1,28 @@
+/*
+ * Ensure there's no duplicated members in each organization
+ */
+
+ function is_candidate() {
+     return this.members && this.members.length > 1;
+ }
+
+ var uniqueMembers;
+ var exist;
+ db.organization.find({'$where': is_candidate}).forEach(org => {
```

Comment:
Can be written `find({members: {$gt: 1}})` and I believe this is faster than using `$where`